instrumentation/kafka/kafka-clients/kafka-clients-0.11/metadata.yaml

```diff
@@ -1,3 +1,6 @@
+description: >
+  This instrumentation provides producer and consumer spans and metrics for Apache Kafka 0.11 clients.
```

Comment:
Maybe replace `producer and consumer spans` with messaging spans. Currently the plan is to disable the kafka metrics by default for 3.0.

Reply:
@laurit I’m leaving the metrics for now, since opentelemetry-java-instrumentation is currently at version 2.x. This should be modified when 3.x is ready.